airbyte-integrations/connectors/source-surveymonkey/unit_tests/test_source.py

```diff
@@ -0,0 +1,28 @@
+#
+# Copyright (c) 2022 Airbyte, Inc., all rights reserved.
+#
+
+from source_surveymonkey.source import SourceSurveymonkey
+
+source_config = {"start_date": "2021-01-01T00:00:00", "access_token": "something"}
+
+
+def test_source_streams():
+    streams = SourceSurveymonkey().streams(config=source_config)
+    assert len(streams) == 4
+
+
+def test_source_check_connection(requests_mock):
+    requests_mock.get(
+        "https://api.surveymonkey.com/v3/users/me", json={"scopes": {"granted": ["responses_read_detail", "surveys_read", "users_read"]}}
```

Comment:
Can we shift url to `conftest.py ` and add it as fixture. The same for `json` response.

Reply:
I was thinking about it ...but decided to leave it here for better understanding/comparison of input/output date